### ISSUE TITLE
Exception on startup

### ISSUE THREAD

**Issue by OMGTallMonster** [michael-lazar/rtv#149]
Since the most recent commit, I get an exception when starting:

```
$ python -m rtv
Connecting...
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/michael/Programming/rtv/rtv/__main__.py", line 98, in <module>
    sys.exit(main())
  File "/home/michael/Programming/rtv/rtv/__main__.py", line 94, in main
    reddit.handler.http.close()
UnboundLocalError: local variable 'reddit' referenced before assignment
```

It looks like there is an exception being thrown during the formatting on line 64 of `__main__.py`. Changing `AGENT.format(__version__)` to `AGENT.format(version=__version__)` seems to fix the issue.

**Comment by michael-lazar** [comment]
Whoops sorry, I committed last night and haven't had a chance to thoroughly test yet! Will fix this when I get home.

**Comment by michael-lazar** [comment]
fixed in 1e3abc17d8965601cb71749f76a965416eac7f8e
